docs/src/components/action/NpmCopyButton.tsx

```diff
@@ -1,68 +1,66 @@
+/* eslint-disable react/prop-types */
 import * as React from 'react';
 import copy from 'clipboard-copy';
-import { Button as ButtonUnstyled, ButtonProps } from '@mui/base/Button';
 import { SxProps } from '@mui/system';
 import { styled, Theme } from '@mui/material/styles';
 import ContentCopyRounded from '@mui/icons-material/ContentCopyRounded';
 import CheckRounded from '@mui/icons-material/CheckRounded';
 
-const Button = styled(ButtonUnstyled)(({ theme }) => ({
+const Button = styled('button')(({ theme }) => ({
```

Comment:
@michaldudak I image the only use case of ButtonUnstyled is when developers create a reusable button that needs to be able to easily support `<div>`, `<span>`, as host elements.